DataFormats/ParticleFlowReco/interface/PFBlockElement.h

```diff
@@ -111,12 +111,25 @@ namespace reco {
     virtual bool isLinkedToDisplacedVertex() const { return false; }
 
     // Glowinski & Gouzevitch
-    void setMultilinks(const PFMultiLinksTC& ml) { multilinks_ = ml; }
-    void setIsValidMultilinks(bool isVal) { multilinks_.isValid = isVal; }
-    void setMultilinksList(const PFMultilinksType& links) { multilinks_.linkedClusters = links; }
-
-    bool isMultilinksValide() const { return multilinks_.isValid; }
-    const PFMultilinksType& getMultilinks() const { return multilinks_.linkedClusters; }
+    void setMultilinks(const PFMultiLinksTC& ml, Type type) { multilinks_[type] = ml; }
+    void setIsValidMultilinks(bool isVal, Type type) { multilinks_[type].isValid = isVal; }
+
+    bool isMultilinksValide(Type type) const {
+      auto it = multilinks_.find(type);
```

Comment:
`const auto&` to avoid a copy.

Reply:
a copy of what? 8 bytes?
a pointer is a pointer is a pointer

Reply:
OK, in this particular case an iterator is returned, it makes no difference, I agree. However, there were a few cases here where potentially a copy of a vector or set would be made.

Reply:
Done.